containers/Investment/RevolvingPool/View/TrancheOverview.tsx

```diff
@@ -38,33 +87,62 @@ const TrancheOverview: React.FC<Props> = (props: Props) => {
           {token} Balance
         </Heading>
         <Heading level="4" margin={{ left: 'auto', top: '0', bottom: '0' }}>
-          1,502.24
+          {addThousandsSeparators(toPrecision(baseToDisplay(balance, 18), 2))}
         </Heading>
       </Box>
 
       <Table>
         <TableBody>
           <TableRow>
             <TableCell scope="row">Current Price</TableCell>
-            <TableCell style={{ textAlign: 'end' }}> 1.232 </TableCell>
+            <TableCell style={{ textAlign: 'end' }}>
+              {' '}
+              {addThousandsSeparators(toPrecision(baseToDisplay(tokenPrice, 27), 2))}{' '}
+            </TableCell>
           </TableRow>
           <TableRow>
             <TableCell scope="row">Your {token} Value</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 1321,523.00</TableCell>
+            <TableCell style={{ textAlign: 'end' }}>
+              DAI {addThousandsSeparators(toPrecision(baseToDisplay(value, 18), 2))}{' '}
+            </TableCell>
           </TableRow>
         </TableBody>
       </Table>
 
-      {card === 'home' && (
-        <Box gap="small" justify="end" direction="row" margin={{ top: 'small' }}>
-          <Button primary label="Redeem" onClick={() => setCard('redeem')} />
-          <Button primary label="Invest" onClick={() => setCard('invest')} />
-        </Box>
+      {isInMemberlist && (
+        <>
+          {card === 'home' && (
+            <Box gap="small" justify="end" direction="row" margin={{ top: 'small' }}>
+              <Button primary label="Redeem" onClick={() => setCard('redeem')} />
+              <Button primary label="Invest" onClick={() => setCard('invest')} />
+            </Box>
+          )}
+          {card === 'order' && (
+            <OrderCard
+              {...props}
+              tinlake={props.tinlake}
+              setCard={setCard}
+              disbursements={disbursements}
+              tokenPrice={tokenPrice}
+            />
+          )}
+          {card === 'collect' && <CollectCard {...props} setCard={setCard} />}
+          {card === 'invest' && <InvestCard {...props} setCard={setCard} />}
+          {card === 'redeem' && <RedeemCard {...props} setCard={setCard} />}
+        </>
+      )}
+
+      {!isInMemberlist && (
+        <Warning>
+          <Heading level="6" margin={{ bottom: 'xsmall' }}>
+            Not allowed
+          </Heading>
+          You are not allowed to invest in the {props.tranche} tranche of {props.pool.name}.
+          <Box justify="end" margin={{ top: 'small' }}>
+            <InvestAction poolName={props.pool.name} />
+          </Box>
+        </Warning>
       )}
```

Comment:
No this will be in the final version. It was missing in the designs, but I discussed it with Dennis. When you are not in the memberlist, we should show a warning/info box and show the same Get started button as on the pool overview.